pkg/provider/k8s/k8s.go

```diff
@@ -1257,6 +1326,34 @@ func (c *K8s) statefulSetReady(resource runtime.Object) (bool, error) {
 	}
 }
 
+func (c *K8s) jobReady(resource runtime.Object) (bool, error) {
+	req := resource.(*batchV1.Job)
+	kind := resource.GetObjectKind().GroupVersionKind().Kind
+	if len(req.Namespace) == 0 {
+		req.Namespace = "default"
+	}
+
+	switch v := resource.GetObjectKind().GroupVersionKind().Version; v {
+	case "v1":
+		client := c.clt.BatchV1().Jobs(req.Namespace)
+
+		res, err := client.Get(req.Name, apiMetaV1.GetOptions{})
+		if err != nil {
+			return false, errors.Wrapf(err, "Checking Job resource:'%v' status failed err:%v", req.Name, err)
+		}
+
+		// current jobReady only works for non-parallel jobs
```

Comment:
not sure how/why is this comment usefull?


And all golang comments should start with a capital letter and end with a full stop

Reply:
currently we only need non-parallel jobs. incase someone tries to add a parallel job someday and prombench fails this comment can be useful. I've not tested this with parallel jobs so I changed the word to `// Current jobReady function only tested with non-parallel jobs.`

added the changes.

Reply:
If that is the case should add it at the top as a comment to this entire function.

Reply:
okay, will add the change